modules/combined/test/tests/reference_residual/tests

```diff
@@ -25,4 +25,31 @@
                   Variables/temp/scaling=0.001'
       prereq = test_sm
     [../]
+    [./group_variables]
+      type = 'Exodiff'
+      input = 'group_variables.i'
+      exodiff = 'group_variables_out.e'
+      abs_zero = 1e-9
+      requirement = 'For ReferenceResidualProblem, groups two standard variables and two scalar variables, respectively.'
+      design = 'ReferenceResidualProblem.md'
+      issues = '#12053'
+    [../]
+    [./group_variables_duplicate_error_check]
+      type = 'RunException'
+      input = 'group_variables.i'
+      expect_err = "A variable cannot be included in multiple groups in the 'group_variables' parameter."
+      cli_args = 'Problem/group_variables="disp_x disp_y; disp_x scalar_strain_zz1"'
+      requirement = 'Test error for ReferenceResidualProblem, variable disp_x is duplicated in the group_variables.'
+      design = 'ReferenceResidualProblem.md'
+      issues = '#12053'
+    [../]
+    [./group_variables_ungroup_error_check]
+      type = 'RunException'
+      input = 'group_variables.i'
+      expect_err = "In the 'group_variables' parameter, variable disp_x is not grouped with other variables."
+      cli_args = 'Problem/group_variables="disp_x; scalar_strain_zz1 scalar_strain_zz2"'
+      requirement = 'Test error for ReferenceResidualProblem, variable disp_x is not grouped with other variable.'
```

Comment:
The reference residual based convergence checking system shall generate an error if a group containing a single variable is defined.